pkg/ebpf/bytecode/tracer-ebpf.go

```diff
@@ -106,7 +106,7 @@ func tracerEbpfDebugO() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "tracer-ebpf-debug.o", size: 59632, mode: os.FileMode(420), modTime: time.Unix(1, 0)}
+	info := bindataFileInfo{name: "tracer-ebpf-debug.o", size: 59632, mode: os.FileMode(436), modTime: time.Unix(1, 0)}
```

Comment:
I often got this change locally. Do we know why this happens?

Reply:
Oooh, hmm, I did not look closely at this, and assumed the size had changed. 

I'm unsure why the permissions would change, but I'll make sure if it should be or not before merging.

Reply:
Seems to be something with file permissions, and did not find root cause, but reverted this change since it probably should not be changed.